docs/guide/apm-data-model.asciidoc

```diff
@@ -90,24 +58,79 @@ Unforseen errors may cause spans to go missing.
 Because the agent notifies the server about how many spans there should be,
 the number of missing spans is able to be calculated and shown in the UI.
 
+[[transactions]]
+=== Transactions
```

Comment:
interesting that you move transactions after spans (so you can say they are a special kind of span, i guess)
nitpick: i'd say extra or additional attributes, not `metadata` (we are overusing a bit this term)

Reply:
@nehaduggal - curious to hear your thoughts on this